models/machine.py

```diff
@@ -57,6 +64,20 @@ def run_filters_and_skip_reason(self, benchmarkable_type, override_filters=None)
 
         machine_run_filters = deepcopy(self.default_filters[benchmarkable_type])
 
+        if (
+            "commit_message_skip_strings" in machine_run_filters
+            # never skip PRs or wheels with otherwise-skippable commit messages
+            and benchmarkable_reason.endswith("-commit")
+            and benchmarkable_commit_msg
+        ):
+            for skip_string in machine_run_filters["commit_message_skip_strings"]:
+                if skip_string in benchmarkable_commit_msg:
```

Comment:
Should these be made case-insensitive? Not sure how consistent commits are with `[MATLAB]` vs `[Matlab]` etc

Reply:
ha, thanks for your analysis up above - it looks like they aren't super consistent. I'll `.upper()` these or something.